app.js

```diff
@@ -120,27 +121,33 @@ function refreshAll(all) {
    var githubPulls;
 
    if (all === true) {
+      debug("rebuilding all pulls");
```

Comment:
Not sure if we want to keep these debug statements around but it is kinda nice keeping it around so I'm indifferent about it